src/couch_multidb_changes.erl

```diff
@@ -93,7 +93,7 @@ handle_call({change, DbName, Change}, _From,
     case {SkipDDocs, is_design_doc(Change)} of
         {true, true} ->
             {reply, ok, State};
-        {false, _} ->
```

Comment:
Covers the case where skip_ddocs is `true` but is_design_doc(Change) is false. Basically we want to call the change callback in every case except `{true, true}`